reana_client/validation.py

```diff
@@ -343,6 +344,20 @@ def _warn_not_defined_parameters(
         )
 
 
+def _validate_cwl_parameters(reana_yaml):
+    """Validate input parameters for CWL workflows.
+
+    :param reana_yaml: REANA YAML specification.
+    """
+    cwl_main_spec_path = reana_yaml["workflow"]["file"]
+    if os.path.exists(cwl_main_spec_path):
```

Comment:
What if the file does not exist, since the user has made a typo? The parameter validator should fail in this case, since the validation cannot be done... I guess the workflow validation would fail before, but if someone calls this function just for the parameter validation purposes, it would be safer to report the error as well.